prisma/migrations/20220113161158_rename_credential_to_installed_app/migration.sql

```diff
@@ -0,0 +1,24 @@
+/*
+  Warnings:
+
+  - You are about to drop the `Credential` table. If the table is not empty, all the data it contains will be lost.
+
+*/
```

Comment:
This is a BIG no-no, we should find a way to rename the table, not drop it and create a new one.

Reply:
🤔 this is the behavior that takes Prisma when we rename the model. I can change it manually but I don't know if we make another change it will put the command for dropping

Reply:
🚨 🚨 🚨 We should prevent this migration to ever reach main 🚨 🚨 🚨 
 We need to modify it and remove the drop statement and find a way to just rename the table and indexes.

Reply:
Might be related [with this](https://github.com/prisma/prisma/issues/7710).